### PR TITLE
Import image

### PR DESCRIPTION
**Pull request by k0da**

### CODE REVIEW

**Comment by kuritka** [comment]
🟥  terratests

**Review by kuritka — DISMISSED** [review]

**Comment by kuritka** [comment]
looks good, thx ! 👍

File: chart/k8gb/templates/operator.yaml [hunk 1]
```diff
@@ -27,7 +27,6 @@ spec:
       containers:
         - name: k8gb
           image: {{ .Values.k8gb.imageRepo }}:{{ .Values.k8gb.imageTag | default .Chart.AppVersion }}
-          imagePullPolicy: Always
```
Comment (ytsarev) [inline chart/k8gb/templates/operator.yaml:30]: Can we please set `imagePullPolicy` explicitly to `IfNotPresent` ?  I know it is default but just to make it explicit

**Review by ytsarev — CHANGES_REQUESTED** [review]

**Review by kuritka — APPROVED** [review]

**Review by ytsarev — APPROVED** [review]
lgtm

**Pull request merged**